include/sway/config.h

```diff
@@ -521,6 +526,7 @@ struct sway_config {
 			char **argv;
 		} leftovers;
 	} handler_context;
+	enum keep_keyboard_layout keep_layout;
```

Comment:
I'd prefer this field to be named `keep_keyboard_layout`. There are other types of layouts, e.g. output layouts.